### PR TITLE
SLING-9504 - Migrate to json

### PR DESCRIPTION
**Pull request by cschneider**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,27 +1,21 @@
...
-    Unless required by applicable law or agreed to in writing,
-    software distributed under the License is distributed on an
-    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-    KIND, either express or implied.  See the License for the
-    specific language governing permissions and limitations
-    under the License.
--->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor 
+    license agreements. See the NOTICE file distributed with this work for additional
```
Comment (tmaret) [inline pom.xml]: It seems the license header got scrambled with the change.

**Review by tmaret — COMMENTED** [review]
* How does it serialise binary data ? 
* Does the code check for message type ? If not, how would we skip unrelated messages in typed consumers ?

File: pom.xml (hunk 1 above)
Comment (cschneider) [reply pom.xml]: Ok . will fix it

Comment (cschneider) [reply pom.xml]: The kafka impl currently does not check message type .. the pipeline one does.

Comment (tmaret) [reply pom.xml]: ack

**Review by tmaret — COMMENTED** [review]
LGTM with the header fix

**Comment by cschneider** [comment]
I just pushed the header fix and the serialization test that shows we are acutally writing bas4 for binaries.

**Pull request merged**